app/views/components/input_date.scala.html

```diff
@@ -0,0 +1,55 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.ui._
+@import controllers.routes._
+@import uk.gov.hmrc.hmrcfrontend.views.html.components._
+
+@(input: models.ui.DateInput, name: String, form: Form[_])(implicit request: Request[_], messages: Messages, ctx: PageContext)
+
+@optionalAria() = @{
+  Seq(input.hint.map(_ => "input-hint"), input.errorMsgs.headOption.map(err => s"${err.id}-error")).flatten match {
+    case Nil => ""
+    case l => ariaDescribedBy(l.mkString(" "))
+  }
+}
+
+<div class="govuk-form-group @if(input.errorMsgs.nonEmpty) {govuk-form-group--error}">
+
+  @if(input.body.isEmpty) {
+      @components.h1_heading_with_label(H1(input.text), name + "-0")
+      @{input.hint.map(hintText => <span id="input-hint" class="govuk-hint">{components.render_text(hintText)}</span>)}
+      @components.error_message(input.errorMsgs)
+      @components.input_date_field(input, name, form)
+
+  } else {
+    @components.h1_heading(H1(input.text))
+    @components.render_components(input.body)
+
+    <fieldset class="govuk-fieldset" @optionalAria()>
```

Comment:
Why is the fieldset element only used when there are additional components on the page?

Reply:
That was what was thrown up in the accessibility review to remove it from single input - but matches the GDS style guide for date input - a fieldset groups fields which are related

Reply:
Ok